src/config/config.ts

```diff
@@ -9,9 +9,9 @@ import {URL, fileURLToPath} from 'node:url'
 import {arch, userInfo as osUserInfo, release, tmpdir, type} from 'node:os'
 import {compact, ensureArgObject, getHomeDir, getPlatform, isProd, requireJson} from '../util'
 import {join, sep} from 'node:path'
-
+// eslint-disable-next-line sort-imports
```

Comment:
Why disable?

Reply:
Try it without.  There's nothing that make it happy

Reply:
ok, now it works happily.  Cool, nevermind.